package.json

```diff
@@ -83,12 +83,5 @@
   "bugs": {
     "url": "https://github.com/sarbbottam/eslint-find-rules/issues"
   },
-  "homepage": "https://github.com/sarbbottam/eslint-find-rules#readme",
-  "xo": {
-    "envs": [
-      "mocha"
```

Comment:
we’ll need to set the mocha env for test files as well.

Reply:
`mocha` is set in the `.eslintrc`, or were you referring to something else?

Reply:
it’s set for every file; i mean it should only apply to tests.